halo2/src/halo2.rs

```diff
@@ -0,0 +1,5825 @@
+#![doc = include_str!("../table.md")]
```

Comment:
This file is missing.

Reply:
Oh yes, we forgot. That file has the protocol description (from the link in the original comment) with some extra annotations. It has a lot of latex math-mode, which requires a [katex](https://katex.org/) header file. I believe it needs to be in the root of the workspace, for `rustdoc` to render the math. Should that be included as well, or would it be more suitable to just remove that line?